src/components/docs/MultiStepSnippet/styles.module.css

```diff
@@ -67,6 +67,19 @@ desktop-breakpoint-query {
     flex: 1;
   }
 
+  .root div[role="tabpanel"] > div {
```

Comment:
<img width="789" alt="Screen Shot 2020-06-14 at 11 55 55 PM" src="https://user-images.githubusercontent.com/20331238/84622553-85ba5800-ae9a-11ea-9b0e-042ebd259266.png">

Reply:
This snippet has overlay, and rather than having it be much longer than the headers, it will now scroll